huxley/api/views/user.py

```diff
@@ -24,7 +24,9 @@ class UserList(generics.ListCreateAPIView):
 
     def get_serializer_class(self):
         if self.request.method == 'POST':
-            return CreateUserSerializer
+            if settings.REGISTRATION_OPEN:
```

Comment:
I'm not convinced that this is the best spot to put this. Instead, we should override the `create` method on this view, and either raise PermissionDenied or call the parent method based on the setting.